package.json

```diff
@@ -27,6 +27,8 @@
   },
   "homepage": "https://ckeditor.com",
   "dependencies": {
+    "ckeditor-plugin-scayt": "https://github.com/WebSpellChecker/ckeditor-plugin-scayt",
+    "ckeditor-plugin-wsc": "https://github.com/WebSpellChecker/ckeditor-plugin-wsc",
```

Comment:
It would be good to have more precise, information, so probably the tag name like `#tag`. From what I see submodules point at latest `5.6.4.0` tag now.